html/includes/common/availability-map.inc.php

```diff
@@ -31,14 +31,28 @@
         $current_width = 10;
     }
 
+    if (isset($widget_settings['show_disabled_and_ignored'])) {
```

Comment:
You need more than isset() here, because it can be set but false. This is how I would write it:
```
$showDisabledAndIgnored = isset($widget_settings['show_disabled_and_ignored']) && $widget_settings['show_disabled_and_ignored'];
```

Then below here you can just `($showDisabledAndIgnored ? 'checked : '')`